ext/gd/tests/gh16322.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+GH-16322 (imageaffine overflow/underflow on affine matrix)
+--EXTENSIONS--
+gd
+--INI--
+memory_limit=-1
+--SKIPIF--
+<?php if (PHP_INT_SIZE != 8) die('skip this test is for 64bit platforms only'); ?>
```

Comment:
The test could make sense on 32bit platforms, too, if instead of `PHP_INT_MIN` and `PHP_INT_MAX` doubles would be used.